solid-python/SolidSolver.py

```diff
@@ -4,12 +4,25 @@
 import sys
 import argparse
 import numpy as np
-import configuration_file as config
-
 import precice
 from precice import *
 
-print("Starting Structure Solver...")
+
+r0 = 1/np.sqrt(np.pi)  # radius of the tube
+a0 = r0**2 * np.pi  # cross sectional area
+tau = 10**10  # timestep size, set it to a large value to enforce tau from precice_config.xml
+N = 100  # number of elements in x direction
+p0 = 0  # pressure at outlet
+L = 10  # length of tube/simulation domain
+E = 10000  # elasticity module
+c_mk = np.sqrt(E/2/r0)  # wave speed
```

Comment:
This appears everywhere as a derived quantity, could you maybe try printing its value and comparing?

Reply:
i checked and its same:)